packages/api/src/external/carequality/api.ts

```diff
@@ -10,13 +10,22 @@ const cqApiMode = Config.isProdEnv()
 
 /**
  * Creates a new instance of the Carequality API client.
- * @param apiKey Optional, API key to use for authentication. If not used, the API key will be retrieved from the environment variables.
  * @returns Carequality API.
  */
-export function makeCarequalityAPI(apiKey?: string): Carequality | undefined {
+export function makeCarequalityAPI(): Carequality | undefined {
```

Comment:
Can we change the name here to reflect that its an API to the CQ Healthcare Directory? I got confused since I was like "doesn't mirth handle sending our requests to other CQ implementers"